int32 label
float64 node
---
bool success
string message
